### PR TITLE
chore(remove_tags transform): Add events to remove_tags transform.

### PR DESCRIPTION
**Pull request by Hoverbear**
Add metrics to remove_events transform, resolving #3403  .

### CODE REVIEW

File: src/internal_events/remove_tags.rs [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+use metrics::counter;
+
+#[derive(Debug)]
+pub struct RemoveTagsEventProcessed;
+
+impl InternalEvent for RemoveTagsEventProcessed {
+    fn emit_metrics(&self) {
+        counter!("events_processed", 1,
+            "component_kind" => "transform",
+            "component_type" => "add_tags",
```
Comment (lukesteensen) [inline src/internal_events/remove_tags.rs:11]:
```suggestion
            "component_type" => "remove_tags",
```

Comment (Hoverbear) [reply src/internal_events/remove_tags.rs:11]: omg.

**Comment by Hoverbear** [comment]
#3606

**Comment by binarylogic** [comment]
@Hoverbear any reason this didn't close #3403?

**Comment by Hoverbear** [comment]
That's for remove_fields! This is remove_tags!

**Comment by binarylogic** [comment]
Ok, look at your PR description. So this should have closed #3404?

**Pull request merged**